### PR TITLE
Enable SSL verification for each of the services mapped to the pods

### PR DESCRIPTION
**Pull request by ccronca**
Add option to run an ssl check on each of the services. The verification is performed by [drwetter/testssl.sh](https://github.com/drwetter/testssl.sh) which connects to the service that is exposed via `oc port-forward <service>`.

### CODE REVIEW

File: pkg/sslchecker/main.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	}
+
+	gateway := n.GetDefaultOutboundIP().String()
+	svc := fmt.Sprintf("%s:%d", c.fqdnSvc(), c.port)
+
+	log.Infof("Starting port forward for service %s:%d in %s:%d", svc, c.port, gateway, localPort)
+
+	portForwardArgs := []string{
+		"port-forward",
+		"--address", gateway,
```
Comment (ccronca) [inline pkg/sslchecker/main.go:70]: This address is used to expose the service to the container.

File: pkg/sslchecker/main.go [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+	testSslArgs := []string{
+		"run", "--rm", "-t",
+		"--add-host", fmt.Sprintf("%s:%s", c.fqdnSvc(), gateway),
+		"-v", fmt.Sprintf("%s:%s:Z", c.hostReportDir, containerReportsDir),
+		"--network", "slirp4netns:allow_host_loopback=true",
+		//@FIXME: quick solution to fix permissions issues with volumes mounted on rootless containers
+		// ref: https://www.redhat.com/sysadmin/debug-rootless-podman-mounted-volumes
+		"--user", "root",
+		"drwetter/testssl.sh",
```
Comment (sfowl) [inline pkg/sslchecker/main.go:112]: suggest putting the full name docker.io/drwetter/testssl.sh here to skip potential short name resolution prompt in podman (I got this when I ran it, which caused it to crash).

File: pkg/sslchecker/main.go [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+		"--jsonfile", c.reportFile(containerReportsDir),
```
Comment (sfowl) [inline pkg/sslchecker/main.go:113]: Perhaps `--quiet` would also be a good option, default drwetter output is rather noisy

**Comment by sfowl** [comment]
Thanks for the contribution. My two comments are very minor, not blockers for merge.

**Pull request merged**